_includes/components/indicator/header.html

```diff
@@ -7,7 +7,10 @@
         </a>
       </div>
       <div class="col-xs-8 col-md-9 col-lg-10">
-        <h1>{{ page.t.general.indicator }} {{ page.indicator.number }}: {{ page.indicator.name }}</h1>
+        <div aria-labelledby="headerIndicatorNumber headerIndicatorName">
```

Comment:
I'm unsure about about the ARIA usage here. Can you describe what the goal is? My understanding is that `aria-labelledby` should be used for associating widget elements with their labels. But this doesn't seem to fall into that category.

Reply:
This is something I have described [here](https://github.com/open-sdg/open-sdg/issues/651#issuecomment-620657544) (second paragraph onwards).

> My understanding is that aria-labelledby should be used for associating widget elements with their labels.

Not exclusively. You can also use it for giving the content of a tag as a label to another. 

The point of this is for the screen readers to read the title tag ("Indicator X.X.X") and the paragraph below (the indicator full name) as one element. This would make it easier for screen reader users to navigate the indicator banner.

Reply:
I'm not sure if it's accomplishing the goal. I tried it out with Orca on Ubuntu - if I do Alt-Shift-H to display a list of headings, it doesn't display this heading at all. If I navigate the headings (using H and Shift-H) it similarly skips that heading. Is it maybe because of the `aria-hidden` attributes?

Not sure Orca is standard enough, but I would assume so. Which screen reader do you use for testing?